config/REST_Config.pm

```diff
@@ -151,10 +151,18 @@ set( 'DOCHAZKA_LDAP_SERVER', 'ldaps://ldap.dochazka.site' );
 #     base DN
 set( 'DOCHAZKA_LDAP_BASE', 'dc=dochazka,dc=site' );
 
-# DOCHAZKA_LDAP_NICK_MAPPING
-#     in order for LDAP authentication to work, the Dochazka 'nick' must
-#     be mapped to a field in the LDAP database (e.g. 'uid', 'cn', etc.)
-set( 'DOCHAZKA_LDAP_NICK_MAPPING', 'uid' );
+# DOCHAZKA_LDAP_MAPPING
+#     in order for LDAP authentication to work, the employee fields that
+#     Dochazka uses, such as 'nick', 'fullname', 'email', etc. must be mapped
+#     to corresponding fields in the LDAP database (e.g. 'uid', 'cn', etc.) -
+#     that is accomplished via this parameter
+#     WARNING: change the values only, never the keys!
+set( 'DOCHAZKA_LDAP_MAPPING', {
+    'nick' => 'uid',
+    'fullname' => '?',
+    'email' => 'email',
+    'sec_id' => 'workforceid',
```

Comment:
for the example purposes, the usual LDAP attributes are:
- nick: uid
- fullname: cn
- email: mail
- sec_id: nothing standard that I know, I would suggest to put there 'work_id'